app/services/guest-cam/index.ts

```diff
@@ -24,6 +24,14 @@ import { ENotificationType } from 'services/notifications';
 import { $t } from 'services/i18n';
 import { JsonrpcService } from 'services/api/jsonrpc';
 import { EStreamingState } from 'services/streaming';
+import Vue from 'vue';
```

Comment:
bit confused about why vue is imported here?

Reply:
This was specifically for a `Vue.set` call I did inside a mutation.  `Vue.set` ensures Vuex reactivity in cases where you are doing something Vuex wouldn't normally recognize as reactive.  i.e. initializing a property on an object that wasn't there when the object was originally created.

Reply:
ah yep! just missed that set call :)